magit-annex.el

```diff
@@ -388,14 +388,15 @@ With a prefix argument, prompt for FILE.
      ,(format "%s FILES.\n\n  git annex %s [ARGS] [FILE...]"
               (capitalize command) command)
      (interactive
-      (list (let ((atpoint (magit-annex-list-file-at-point)))
+      (list (let* ((selectedfiles (magit-annex-list-files-at-point-or-region))
+		   (default (and selectedfiles (mapconcat #'identity selectedfiles ","))))
```

Comment:
Please avoid tabs.  (Sorry, I should make that preference clear with an entry in .dir-locals.el.)